qiita_pet/handlers/websocket_handlers.py

```diff
@@ -70,3 +72,26 @@ def on_close(self):
         yield Task(self.toredis.unsubscribe, self.channel)
         self.r_client.delete('%s:messages' % self.channel)
         self.redis.disconnect()
+
+
+class SelectSamplesHandler(WebSocketHandler, BaseHandler):
+    @authenticated
+    def on_message(self, msg):
```

Comment:
We don't require them, and we have been doing them sparsely. However I
think they are always worth-it to have. In this case likely describing
the context where the messages come from.

On (Apr-15-15| 6:38), Joshua Shorenstein wrote:
> > @@ -70,3 +72,26 @@ def on_close(self):
> >          yield Task(self.toredis.unsubscribe, self.channel)
> >          self.r_client.delete('%s:messages' % self.channel)
> >          self.redis.disconnect()
> > +
> > +
> > +class SelectSamplesHandler(WebSocketHandler, BaseHandler):
> > +    @authenticated
> > +    def on_message(self, msg):
> 
> Are we requiring docstrings in qiita_pet on tornado built-in functions? So far we haven't, but we are re-writing the style guidelines so should have the discussion.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/1056/files#r28419274